fixbaserom.py

```diff
@@ -7,10 +7,10 @@
 
 # Determine if we have a ROM file
 romFileName = ""
-if (path.exists("baserom_original.z64")):
-    romFileName = "baserom_original.z64"
-elif (path.exists("baserom_original.n64")):
-    romFileName = "baserom_original.n64"
+if (path.exists("baserom.z64")):
+    romFileName = "baserom.z64"
+elif (path.exists("baserom.n64")):
+    romFileName = "baserom.n64"
```

Comment:
Won't this change make it so it takes the existing baserom.z64 and override it? I don't understand why we would want to do that.
If we ask users to provide a baserom.z64, we should either use it as is or create a completely new rom that will be used as the base for extraction. Silently modifying it is very bad practice imo, and I would expect many users to be at least confused by it.